client/components/TestReview/index.jsx

```diff
@@ -0,0 +1,270 @@
+                heading="Tests for Test Plan Version"
+            />
+        );
+    }
+
+    if (error || !data?.testPlanVersion) {
+        const errorMessage =
+            error?.message ??
+            `Failed to find a test plan version with ID ${testPlanVersionId}`;
+        return (
+            <PageStatus
+                title="Tests for Test Plan Version | ARIA-AT"
+                heading="Tests for Test Plan Version"
+                message={errorMessage}
+                isError
+            />
+        );
+    }
+
+    const testPlanVersion = data.testPlanVersion;
+    const testCount = testPlanVersion.tests.length;
+    const atNames = unique(
+        testPlanVersion.tests.flatMap(test => test.ats.map(at => at.name))
+    );
+
+    let filteredTests;
+    if (activeFilter === 'All ATs') {
+        filteredTests = testPlanVersion.tests;
+    } else {
+        filteredTests = testPlanVersion.tests.filter(test =>
+            test.ats.find(at => at.name === activeFilter)
+        );
+    }
+
+    const filterOptions = Object.fromEntries(
+        ['All ATs', ...atNames].map(key => {
+            let count;
+            if (key === 'All ATs') {
+                count = testPlanVersion.tests.length;
+            } else {
+                count = testPlanVersion.tests.filter(test =>
+                    test.ats.find(at => at.name === key)
+                ).length;
+            }
+            return [key, `${key} (${count})`];
+        })
+    );
+
+    return (
+        <Container id="main" as="main" tabIndex="-1">
+            <Helmet>
+                <title>{`Tests for ${testPlanVersion.title} ${testPlanVersion.versionString} | ARIA-AT`}</title>
+            </Helmet>
+            <h1>
+                {`Tests for ${testPlanVersion.title} ` +
+                    `${testPlanVersion.versionString}` +
+                    `${testPlanVersion.deprecatedAt ? ' (Deprecated)' : ''}`}
+            </h1>
+            <h2>Introduction</h2>
+            <p>
+                {`This page contains the full content of all ${testCount} ` +
+                    `tests the ARIA-AT team has authored for ` +
+                    `${testPlanVersion.title} as of version ` +
+                    `${testPlanVersion.versionString}.`}
+            </p>
+            <h2>Metadata</h2>
+            <ul>
+                <li>
+                    <strong>Phase:&nbsp;</strong>
+                    {derivePhaseName(testPlanVersion.phase)}
+                </li>
+                <li>
+                    <strong>Version:&nbsp;</strong>
+                    {`${testPlanVersion.versionString} `}
+                    <Link
+                        to={`/data-management/${testPlanVersion.testPlan.directory}`}
+                    >
+                        (View all versions of this test plan)
+                    </Link>
+                </li>
+                <li>
+                    <strong>Version History:&nbsp;</strong>
+                    <Ul>
```

Comment:
This doesn't cover the following condition from #788:

> If current phase is R&D Complete, add list item with string "This version is a proposed draft; ARIA-AT review processes have not yet started."

Reply:
Ah right, I actually noticed a contradiction in between the spec and the example so I decided to go with the text the example gave, which is "R&D completed on Apr 14, 2022 for this version." Do you think that's ok?